### PR TITLE
Feature/impersonate user profiler

### PR DESCRIPTION
**Pull request by Backins**
Feature ready. I need authentification to test my feature. But back and front are ready to merge.

### CODE REVIEW

**Review by ivanPrunier — APPROVED** [review]

**Comment by ivanPrunier** [comment]
I approved but can you try to fix Codecy problems before merging?

**Comment by Backins** [comment]
When I delete the unused variable, Symfony crash. And I don't understand the two others. Can you describe the problem ?

**Comment by Backins** [comment]
Miss click "Close and comment" :(

**Review by mehdou92 — APPROVED** [review]

**Comment by Backins** [comment]
With authentification by @mehdou92, impersonate an user is now ready. 

NB : You can't impersonate to a second user, you need exit the first impersonation. I can't fixed the issue and I think it's too expansive to fix it.

Wait and see your comments.

File: templates/data_collector/template.html.twig [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    let button = document.getElementById('impersonate-button');
+                    let parsedHref = location.href.split('?')[0];
+
+                    button.onclick = function () {
+                        location.href = parsedHref + '?_switch_user=' + user.value;
+                    };
+                </script>
+            {% else %}
+                <b>User</b>
+                <span>0 users founded</span>
```
Comment (ivanPrunier) [inline templates/data_collector/template.html.twig]: found* 😉

**Review by ivanPrunier — APPROVED** [review]

File: templates/data_collector/template.html.twig (hunk 1 above)
Comment (Backins) [reply templates/data_collector/template.html.twig]: Fixed 💃

**Review by benKazmierczak — APPROVED** [review]

**Review by jeremNoel — APPROVED** [review]

**Pull request merged**